src/OSPSuite.Presentation/Presenters/SimulationVsObservedDataChartPresenter.cs

```diff
@@ -85,5 +87,11 @@ protected void AddUsedObservedDataToChart()
       }
 
       protected abstract void AddRunResultToChart();
+      
+      public void Handle(SimulationOutputMappingsChangedEvent eventToHandle)
+      {
+         if (eventToHandle.Simulation == _simulation)
```

Comment:
this can cras if Simulation is null (I think). so we usually do (Equals()) .
I may be wrong

Reply:
I'll change it to Equals, but I don't think it should crash. if one is null

Reply:
Ok . Maybe old behavior then